wbooks/src/app/screens/book-detail/styles.js

```diff
@@ -0,0 +1,103 @@
+import { StyleSheet } from 'react-native';
+
+import { cyan, lightGray, red, white, lightBlue, lightGreen } from '../../../constants/colors';
+
+export default StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: cyan
+  },
+  cardDetail: {	
+    borderRadius: 5,
+    backgroundColor: white,
+    margin: 20
+  },
+  cardContainer: {
+    flexDirection: 'row',
+    borderRadius: 5,
+    marginStart: 30,
+    marginTop: 8,
+    marginEnd: 30
```

Comment:
marginRight

Reply:
Also if you have the same value on `marginLeft` and `marginRight` you can replace both with `marginHorizontal`

Reply:
great!